rest-server/admin/src/main/scala/com/actionml/admin/MongoAdministrator.scala

```diff
@@ -155,8 +155,6 @@ class MongoAdministrator extends Administrator with JsonParser {
       }
     } else {
       logger.trace("Getting status for all Engines")
-      val statuses = jsonList(
-        engines.mapValues(_.status()).map(_._2.getOrElse(jsonComment(s"Warning: no status returned"))).toSeq)
```

Comment:
I don't think removing this is needed or good. This is where the status of engines is merged for display in the CLI

Reply:
What does removing this solve?

Reply:
@pferrel this variable is never used